mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -123,7 +123,17 @@ def test_selvals(self):
             ['time_avg_avgValueWithinOceanLayerRegion_avgLayerTemperature',
              'refBottomDepth']
 
-        selvals = {'nVertLevels': 0}
+        vertIndex = 10
```

Comment:
I just split the difference in my change and forced testing over the fully range of 0...10